src/app/Dashboard/LayoutTemplatePicker.tsx

```diff
@@ -0,0 +1,578 @@
+              {selectedTemplate?.vendor && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Vendor</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.vendor}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              {selectedTemplate?.version && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Version</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.version}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              <DescriptionListGroup>
+                <DescriptionListTerm>Preview</DescriptionListTerm>
+                {
+                  <div className="dashboard-layout-preview">
+                    <ServiceContext.Provider value={fakeServices}>
+                      <ChartContext.Provider value={fakeChartContext}>
+                        <Grid
+                          id={'dashboard-layout-preview-grid'}
+                          hasGutter
+                          onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                          }}
+                        >
+                          {selectedTemplate.cards
+                            .slice(0, CARD_PREVIEW_LIMIT)
+                            .filter((cfg) => hasConfigByName(cfg.name))
+                            .map((cfg, idx) => (
+                              <FeatureFlag level={getConfigByName(cfg.name).featureLevel} key={`${idx}-wrapper`}>
+                                <GridItem span={cfg.span} key={idx} order={{ default: idx.toString() }}>
+                                  {/* TODO: remove this once we have a preview for JFRMetricsChartCard */}
+                                  {cfg.name === JFRMetricsChartCardDescriptor.component.name ? (
+                                    <Card isFullHeight isCompact>
+                                      Empty preview
+                                    </Card>
+                                  ) : (
+                                    React.createElement(getConfigByName(cfg.name).component, {
+                                      span: cfg.span,
+                                      ...cfg.props,
+                                      dashboardId: idx,
+                                      actions: [],
+                                    })
+                                  )}
+                                </GridItem>
+                              </FeatureFlag>
+                            ))}
+                        </Grid>
+                      </ChartContext.Provider>
+                    </ServiceContext.Provider>
+                  </div>
+                }
+              </DescriptionListGroup>
+            </DescriptionList>
+          ) : (
+            <EmptyState variant={EmptyStateVariant.full}>
```

Comment:
Wrap this in Bulleye? EmptyState just centers on x-axis. Bulleye should expand all dimension and put child in center.

![Screenshot from 2023-04-11 21-52-52](https://user-images.githubusercontent.com/68053619/231326955-e4110b51-3f09-4c38-97a4-d6e1e8132465.png)